src/payment/checkout/Checkout.jsx

```diff
@@ -182,6 +182,15 @@ class Checkout extends React.Component {
     const shouldDisplayStripePaymentForm = !loading && enableStripePaymentProcessor && options.clientSecret;
     const shouldDisplayCyberSourcePaymentForm = !loading && !enableStripePaymentProcessor;
 
+    let stripePromise;
+    if (shouldDisplayStripePaymentForm) {
+      stripePromise = loadStripe(process.env.STRIPE_PUBLISHABLE_KEY, {
```

Comment:
In addition to @julianajlk's point above, I think the reason we originally put `loadStripe` outside the component was to follow [this Stripe doc](https://stripe.com/docs/payments/quickstart#load-stripe):

> Make sure to call loadStripe outside of a component’s render to avoid recreating the Stripe object on every render.

One alternative would be to update the Stripe options after the i18n config is available. That looks tricky though, because per the [Stripe React docs on `options`](https://stripe.com/docs/stripe-js/react#elements-provider), it looks like `options` cannot be changed after it is set.

I suspect that the Stripe `options` would need to be strategically loaded after the i18n config is available, but before the component tree is rendered. I also suspect we load our i18n config on render, which is why this is tricky.

All this to say: this might be worth a comment as to why we are deviating from the Stripe docs. And we should keep an eye on this page's performance as we are re-loading Stripe with every render.

Reply:
Added a comment in the code here. Per our discussion we can merge this and examine performance on stage. My guess is this maaay end up be an issue (thank you for identifying), but we'll see.